packages/web-components/src/global/_variables.scss

```diff
@@ -37,6 +37,7 @@ $font-weight-normal: 400;
 $font-weight-bold: 700;
 $font-style-normal: normal;
 $font-style-italic: italic;
+$base-font-size: 16px;
```

Comment:
Can we add an inline comment to this (or rename) that mentions that this is the base font size for USWDS v3? It's a little confusing because the Formation base font size is something different.

Reply:
I see what you mean, as that's the base font size for USWDS v3 itself. ;)